### PR TITLE
20159 - Add view_address permission to admin account

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/20159

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by seeker25** [comment]
Should we update package.json and package-lock?

File: auth-api/migrations/versions/2024_05_30_39b4f331c003_admin_view_address_permission.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+
+def upgrade():
+    op.execute(
+        "INSERT INTO permissions (id, membership_type_code, actions) VALUES (84, 'ADMIN', 'view_address')"
+    )
+
+
+def downgrade():
+    op.execute('DELETE FROM permissions WHERE id=84')
```
Comment (seeker25) [inline auth-api/migrations/versions/2024_05_30_39b4f331c003_admin_view_address_permission.py:26]: This isn't right

**Review by seeker25 — CHANGES_REQUESTED** [review]
migration needs to be fixed

File: auth-api/migrations/versions/2024_05_30_39b4f331c003_admin_view_address_permission.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# revision identifiers, used by Alembic.
+revision = '39b4f331c003'
+down_revision = '41fa6588c76e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        "INSERT INTO permissions (id, membership_type_code, actions) VALUES (84, 'ADMIN', 'view_address')"
```
Comment (seeker25) [inline auth-api/migrations/versions/2024_05_30_39b4f331c003_admin_view_address_permission.py:21]: I wouldn't be including ids, this should be automatic

File: auth-api/migrations/versions/2024_05_30_39b4f331c003_admin_view_address_permission.py (hunk 1 above)
Comment (seeker25) [reply auth-api/migrations/versions/2024_05_30_39b4f331c003_admin_view_address_permission.py:26]: `DELETE from permissions where actions = 'view_address'`

File: auth-api/migrations/versions/2024_05_30_39b4f331c003_admin_view_address_permission.py (hunk 2 above)
Comment (Jxio) [reply auth-api/migrations/versions/2024_05_30_39b4f331c003_admin_view_address_permission.py]: yup updated

**Review by seeker25 — APPROVED** [review]

**Pull request merged**